.github/workflows/python-app.yml

```diff
@@ -14,8 +14,11 @@ permissions:
 
 jobs:
   build:
-
-    runs-on: self-hosted
```

Comment:
Thanks for the PR. Well, we do self-hosted for a reason. PTAL here: https://github.com/google/gcp_scanner/wiki/GCP-Scanner-Development-and-Testing#acceptance-testing

Reply:
I didn’t quite understand the drawbacks of my approach, so is it better to have the Mac OS and Windows self hosted on a GCP VM

Reply:
and is that the reason why the tests are failling?

Reply:
PTAL at the link I provided we need self-hosted infra to check GCP scanner against a live GCP project. You can also study `test_acceptance.py` for our approach here.

Reply:
Ok thank you so much for the guidance and your time I will look more into it and work on making it run on multiple self-hosted VMs